ci-operator/config/openshift-psap/ci-artifacts/openshift-psap-ci-artifacts-release-4.8__nfd.yaml

```diff
@@ -7,7 +7,7 @@ build_root:
   image_stream_tag:
     name: release
     namespace: openshift
-    tag: golang-1.16
+    tag: golang-1.17
```

Comment:
No, the branch 4.8 should be tested with 1.16 
https://github.com/openshift/cluster-nfd-operator/blob/release-4.8/go.mod#L3

Reply:
fixed